app/server.js

```diff
@@ -115,7 +118,13 @@ DB.connect(config.DBHOST, dbAuth).then(async () => {
   apiRoutes(app, http);
   serverRenderingRoutes(app);
   if (config.sentry.dsn) {
-    app.use(Sentry.Handlers.errorHandler());
+    app.use(
+      Sentry.Handlers.errorHandler({
+        shouldHandleError(error) {
+          return error.status >= 500;
```

Comment:
@elreplicante We do not have status in our errors i think ? there is two type of errors we want to handle that i can think of, either is one of our errors which has ```code``` property not status, the other one is an instance of Error, something like:
```return error instanceof Error || error.code >= 500```